src/main/java/de/doubleslash/keeptime/model/Project.java

```diff
@@ -37,6 +23,7 @@ public class Project {
    @Column(name = "id", updatable = false, nullable = false)
    private long id;
 
+   @NotEmpty(message = "Name must not be null or empty")

```

Comment:
Such added annotations should now be moved to our DTO objects. As I assume these should validate what we receive via REST-API.